whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -321,6 +289,22 @@ private static Autnum createAutnumResponse(final RpslObject rpslObject) {
         return autnum;
     }
 
+    private static Autnum createAsBlockResponse(final RpslObject rpslObject) {
+        final Autnum autnum = new Autnum();
+
+        String key = rpslObject.getValueForAttribute(AttributeType.AS_BLOCK).toString();
+        String[] startAndEndKey = key.split("-");
+
+        autnum.setHandle(startAndEndKey[0]);
+        //TODO :check what should be the name
+        autnum.setName(key.replace(" ", ""));
+        autnum.setStartAutnum(Long.parseLong(startAndEndKey[0].trim().substring(2)));
+        autnum.setEndAutnum(Long.parseLong(startAndEndKey[1].trim().substring(2)));
+        autnum.setType("DIRECT ALLOCATION");
```

Comment:
Where did you find 'DIRECT ALLOCATION' defined? Also, define as a static field.

Reply:
We already send the hardcoded value  'DIRECT ALLOCATION' for autnum response, I used the same value (check function createAutnumResponse). Although specification says "type -- a string containing an RIR-specific classification of the autnum". not sure if it correct, we can discuss on this.

Reply:
Ok! I see it elsewhere in the code. We can leave as-is for now.